requirements/edx/base.txt

```diff
@@ -91,6 +91,7 @@ pysrt==0.4.7
 PyYAML==3.10
 requests==2.7.0
 requests-oauthlib==0.4.1
+rpdb==0.1.6
```

Comment:
@efischer19 I'd prefer to see this introduced in its own PR. Also, it should go in development.txt instead of base if it is only needed for development.